### DISCUSSION TITLE
Using Auto Complete

### DISCUSSION THREAD

**Discussion by WeirDave** [phcode-dev/phoenix#1737] [Q&A] [answered]
I can't seem to figure out how to use the auto complete feature and select the choices available. I have tried, enter, space and tab. I must be overlooking the obvious but I can't seem to find it. Thank you in advance.

**Comment by abose** [comment, marked as answer]
* press `ctrl-space` (`Cmd-space` in mac) to bing up auto-complete options.
* press up and down arrow keys to select the desired code hint
* press enter to commit.
* Auto-complete works in js, HTML(and related like htm) and CSS(LESS/SCSS) files.

Is this what you are looking for?

**Reply by WeirDave** [reply]
Thank you! I kept seeing it auto pop up but it never stayed up. Calling it up myself with ctrl-space seems to work and enter to select
